app/models/user.rb

```diff
@@ -104,11 +109,19 @@ def build_namespace_path
   end
 
   def ensure_namespace
+    return unless User.user_types[user_type] == User.user_types[:human]
```

Comment:
**nitpick (blocking):** Same as above, I think this can be `return unless user.human?`

Reply:
Updated in 8929d48319a2c018f27c667dec548413e9eb6af2